CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.0.5 - 2019-05-20
+### Changed
+- Move, Modify and Delete tools combined to a single tool.
+
 ## 0.0.4 - 2019-03-11
```

Comment:
0.0.4 has not been publicly released . According to https://keepachangelog.com/en/1.0.0/ we should use the unreleased section.
```suggestion
## [unreleased]
```